towhee/engine/thread_pool_task_executor.py

```diff
@@ -46,6 +45,7 @@ def __init__(self, name: str, cache_path: str = None):
         self._thread_pool = ThreadPoolExecutor()
         self._is_run = True
         self.setDaemon(True)
+        self._framework = 'pytorch'
```

Comment:
Why does the executor have framework info? It's only the pipeline or operator's attribute, not the executor's.